.github/workflows/frontend_lint.yaml

```diff
@@ -66,3 +67,17 @@ jobs:
         cd components/crud-web-apps/volumes/frontend
         npm i
         npm run lint-check
+
+  lint-centraldashboard-angular-code:
```

Comment:
I'd propose to drop the linting checks for the dashboard, until we fully migrate to eslint